Contract Extension Flow (#166)
ConsenSys/doc.goquorum: docs/concepts/privacy/contract-extension.md

If the network is running with [enhanced network permissioning](../permissions-overview.md#enhanced-network-permissioning),
only a network or organization administrator can initiate or accept a contract extension.

## Flow

The following diagram describes the flow of contract state extension.
![contract state extension diagram](../../images/ContractStateExtension.png)

> In this example, an existing private contract deployed between Nodes A and B is being extended to Nodes C from Node A.

Reviewer: ```suggestion
In this example, Node A extends an existing private contract to Nodes C that was initially deployed only between Nodes A and B.
```
